Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby:2.3-alpine
+
+ENV DOCKER_VERSION=1.7.1
+
+RUN apk add --update curl bash grep && \
+  curl -sSL https://get.docker.com/builds/Linux/x86_64/docker-$DOCKER_VERSION -o /usr/bin/docker && \
```

Comment:
use full name for the arguments when possible (`--output` instead of the `-o`, ...)